src/views/settings/view_protection_settings.cpp

```diff
@@ -8,10 +8,13 @@ namespace big
 			*(bool*)i = state;
 	}
 
-	void view::protection_settings()
+	static inline void reset_protections()
 	{
-		auto initial_protections = g.protections;
+		g.protections = {};
```

Comment:
Won't this just zero out the protections struct?

Reply:
Nope that should default construct the object, as long as every field has a value it'll set it correctly.

Reply:
https://godbolt.org/z/49oqszh63